tokio-net/src/util/poll_evented.rs

```diff
@@ -168,28 +166,13 @@ where
     E: Evented,
 {
     /// Creates a new `PollEvented` associated with the default reactor.
-    pub fn new(io: E) -> PollEvented<E> {
-        PollEvented {
-            io: Some(io),
-            inner: Inner {
-                registration: Registration::new(),
-                read_readiness: AtomicUsize::new(0),
-                write_readiness: AtomicUsize::new(0),
-            },
-        }
+    pub fn new(io: E) -> io::Result<Self> {
+        Self::new2(io, None)
     }
 
     /// Creates a new `PollEvented` associated with the specified reactor.
     pub fn new_with_handle(io: E, handle: &Handle) -> io::Result<Self> {
```

Comment:
Again, if `Handle::current` is the only way to get a `Handle` now, there's probably no reason to continue providing this function...

Reply:
This has been removed.